docs/projects/fps/tutorial.md

```diff
@@ -141,12 +142,7 @@ For health packs we will do the latter, so that when the game begins there will
 
 #### Editing snapshot generation
 
-The SpatialOS menu option in the Unity editor include an item **"Generate FPS Snapshot"**. This runs the script `Assets/Fps/Scripts/Editor/SnapshotGenerator/SnapshotMenu.cs`, which you can find from within your Unity editor.
-
-We will modify the snapshot generating logic in two steps:
```

Comment:
are we not donig this anymore?

Reply:
No, the current state of master doesn't require you to change import settings anymore. I felt this part was worth removing because it talks about a very advanced topic which feels didn't relevant for what this tutorial is trying to teach.